spec/latest/core/core.jsonld

```diff
@@ -165,14 +165,14 @@
       "@type": "hydra:Link",
       "label": "supported properties",
       "comment": "The properties known to be supported by a Hydra class",
-      "domain": "hydra:Class",
+      "domain": "rdfs:Class",
       "range": "hydra:SupportedProperty",
       "vs:term_status": "testing"
     },
     {
       "@id": "hydra:SupportedProperty",
       "@type": "hydra:Class",
-      "subClassOf": "hydra:Resource",
+      "subClassOf": "rdfs:Resource",

```

Comment:
I think this is now superfluous and can be removed. `rdfs:Resource` will be the implicit super class